spec/models/claim_spec.rb

```diff
@@ -53,6 +53,11 @@
     it { is_expected.to delegate_method(:full_name).to(:submitted_by).with_prefix.allow_nil }
   end
 
+  context "with auditing" do
+    it { is_expected.to be_audited }
+    it { is_expected.to have_associated_audits }
+  end
```

Comment:
I think 'describe' is the more appropriate method to use here. Then Rubocop won't nag you to start your name with "with/when/where/etc."

```suggestion
  describe "auditing" do
    it { is_expected.to be_audited }
    it { is_expected.to have_associated_audits }
  end
```